### DISCUSSION TITLE
Darstellungsfilter Auswahlliste mit Abhängigkeit

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#190] [General]
Hallo,
ist es möglich einen Darstellungsfilter mit Auswahlliste so zu konfigurieren -  mit zwei "Key-Fields", in dem die zweite Auswahlliste abhängig von dem ist, was in der ersten Auswahlliste ausgesucht wurde.

Bsp: 
|Darstellungsfilter AB|
|-|
|**Type** |
| A oder B |
| **Subtype** |
| x,y,z oder u,v,w |

Type A hat Subtypes x, y, z
Type B hat Subtypes u, v, w

Je nachdem welchen **Typ** man auswählt erhält man unterschiedliche Auswahllisten für **Subtyp**.

**Comment by gis-eni** [comment]
Derzeit ist es nicht möglich, eine abhängige Auswahlliste bei den Darstellungsfiltern zum konfiguriren. Eine Implementierung ist aufwendig und kommt nur wenn GANZ WICHTIG!

**Comment by gis-eni** [comment]
Eventuell wäre es besser die UI in den TOC zu verlegen:
Filtersymbol beim Thema => UI ...
